src/tests/test_db_create_handler.py

```diff
@@ -144,6 +144,17 @@ def test_restore_db_cluster(self, mock_rds, mock_secrets_client):
             "SecretString": my_secret_string
         }
         mock_secrets_client.get_secret_value.return_value = mock_secret_payload
+        two_days_ago = datetime.datetime.now() - datetime.timedelta(2)
+        date_str = _get_date_string(two_days_ago)
+        mock_rds.describe_db_snapshots.return_value = {
+            'DBSnapshots': [
+                {
+                    "DBInstanceIdentifier": 'aqts-capture-db-legacy-production-external',
+                    "DBSnapshotIdentifier": f"rds:aqts-capture-db-legacy-production-external-{date_str}"
```

Comment:
May want to shove `aqts-capture-db-legacy-production-external` into a variable you can sub in on these 2 lines.